docker/docker-compose.dev.yml

```diff
@@ -50,3 +51,12 @@ services:
       MB_IMPORT_DUMPS: "true"
     ports:
       - "5430:5432"
+
+  static_builder:
+    build:
+      context: ..
+      dockerfile: Dockerfile.webpack
+    command: npm run dev
+    volumes:
+      - ..:/code:z
+      - ../critiquebrainz/frontend/static:/static:z
```

Comment:
why is this needed? I don't see a reference to `/static` in the webpack configuration file, so I don't think it has any effect

Reply:
Done!